lib/QubitCsvImport.class.php

```diff
@@ -80,42 +90,74 @@ public function import($csvFile, $type = null, $csvOrigFileName = null)
         break;
     }
 
-    // Figure out whether indexing flag should be added to command
+    // Figure out whether indexing flag should be added to command.
     $commandIndexFlag = ($taskClassName != 'csv:event-import' && $this->indexDuringImport) ? '--index' : '';
 
-    // Build command string
+    if ('' !== $this->updateType)
+    {
+      switch ($this->updateType)
+      {
+        case 'import-as-new':
+          $commandSkipMatched = ($this->skipMatched) ? '--skip-matched' : '';
+          break;
+
+        case 'match-and-update':
+        case 'delete-and-replace':
+          $commandUpdate = ($this->updateType == 'match-and-update') ? '--update="match-and-update"' : '--update="delete-and-replace"';
+          $commandSkipUnmatched = ($this->skipUnmatched) ? '--skip-unmatched' : '';
+          $commandLimit = ('' !== $this->limit) ? "--limit=\"$this->limit\"" : '';
```

Comment:
Changing to single quotes as you have there will print the literal: **--limit="$this->limit"**
as opposed to evaluating $this->limit: **--limit="museum-of-agriculture"**